### PR TITLE
Adding a unit test that would gate PRs and prevent reverts, e.g. #83327

### PR DESCRIPTION
**Pull request by weiwangmeta**
PR #83327 slipped through CI, adding this unit test as part of efforts to minimize future reverts

### CODE REVIEW

File: test/test_linalg.py [hunk 1]
```diff
@@ -7453,6 +7453,11 @@ def test_preferred_linalg_library(self):
         self.assertEqual(out_ref, out1.cpu())
         self.assertEqual(out1, out2)
 
+    def test_permute_matmual(self):
```
Comment (Balandat) [inline test/test_linalg.py:7456]:
```suggestion
    def test_permute_matmul(self):
```

**Review by Balandat — APPROVED** [review]

File: test/test_linalg.py (hunk 1 above)
Comment (weiwangmeta) [reply test/test_linalg.py]: Good catch, thank you!

Comment (Balandat) [reply test/test_linalg.py]: Yeah this happened to me on mualtiple occasions

Comment (weiwangmeta) [reply test/test_linalg.py]: haha, just got your huamor :)

**Review by mehtanirav — APPROVED** [review]

**Pull request closed without merging**